### ISSUE TITLE
Running the auto generation demo taking too long

### ISSUE THREAD

**Issue by ksai2324** [UX-Decoder/Semantic-SAM#60]
I'm trying to run the auto generation demo on a T4 GPU (16 virtual memory), but it is taking too long (more than one hour), is that a normal behavior? How much GPU did you use for inference? 

Thanks in advance!

**Comment by FengLi-ust** [comment]
Yes, the GPU is not good enough for use. You can run a local demo for fast inference. Thanks!
